modules/data.atmosphere/DESCRIPTION

```diff
@@ -52,6 +52,7 @@ Imports:
     truncnorm,
     udunits2 (>= 0.11),
     XML (>= 3.98-1.4),
+    xts,
```

Comment:
You will probably need to run `scripts/generate_dependencies.R` to update the dependency list for Travis, Docker, etc.

Also, can you add it to the list of binary packages installed by Travis? It should be `r-cran-xts`.

Reply:
> Also, can you add it to the list of binary packages installed by Travis? It should be r-cran-xts.
how am I supposed to do that ?

Reply:
Add `r-cran-xts` to [this list in .travis.yml](https://github.com/PecanProject/pecan/blob/develop/.travis.yml#L65-L91).